components/admin/lessons/AdminLessonsSideBar.tsx

```diff
@@ -13,14 +14,17 @@ export const AdminLessonsSideBar: React.FC<SideBarLessonProps> = ({
   setSelectedLesson,
   selectedLesson
 }) => {
-  const lessonListData = lessons || []
-
+  const lessonListData = _.cloneDeep(lessons) || []
```

Comment:
Without cloning it this function tries to edit useState value which will cause runtime typescript errors.